### ISSUE TITLE
[hi] Localize `content/en/case-studies/_index.html` to Hindi

### ISSUE THREAD

**Issue by anubha-v-ardhan** [kubernetes/website#30731]
**This is a Feature Request**

**What would you like to be added**
We need to localize [content/en/case-studies/_index.html](https://github.com/kubernetes/website/blob/main/content/en/case-studies/_index.html) to Hindi (`content/hi/case-studies/_index.md`)

**Why is this needed**
This page is needed as per [minimum required content](https://kubernetes.io/docs/contribute/localization/#minimum-required-content) for the ongoing Hindi Localization effort.

**Proposed Solution**
Something like:
```
---
title: <Hindi Translation>
linkTitle: <Hindi Translation>
bigheader: <Hindi Translation>
abstract: <Hindi Translation>
layout: basic
class: gridPage
cid: caseStudies
---
```
**Note**: 
1. To know about the preferred localization of common Kubernetes related terms, check this [spreadsheet](https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0)
2. Please join the Kubernetes Slack (via https://slack.k8s.io/ if you haven't already) and dedicated channel for Hindi localization [#kubernetes-docs-hi](https://kubernetes.slack.com/archives/CJ14B9BDJ) for discussions

**Comments**
The current working branch for `hi` localization work is `dev-1.22-hi.1`. Please make sure to open a PR against `dev-1.22-hi.1`.(Starting work by branching off `dev-1.22-hi.1` instead of `main` is a good idea)
For more info see Hindi localization progress tracking issue #29353 

/languge hi
/kind feature
/good-first-issue

**Comment by harshmathurx** [comment]
Hey I would like to work on this

**Comment by anubha-v-ardhan** [comment]
/close
Fixed with #30733
